### PR TITLE
Feature/add access media location permission

### PR DESCRIPTION
**Pull request by hectoruiz**
# Summary

The permission is needed to access location tags of media: https://developer.android.com/training/data-storage/shared/media#media-location-permission

## Test Plan

These modifications were tested in a react native app using the patch-package library to install it and have worked.
These outputs show it:
(Real device)
2021-12-02 22:47:45.263 22013-7798/com.blabla.blabla I/ReactNativeJS: { permissions: 
       { 'android.permission.READ_EXTERNAL_STORAGE': 'granted',
         'android.permission.ACCESS_FINE_LOCATION': 'granted',
         'android.permission.ACCESS_MEDIA_LOCATION': 'granted' } }

(Emulator)
2021-12-02 23:02:03.719 3039-3238/com.blabla.blabla I/ReactNativeJS: { permissions: 
       { 'android.permission.ACCESS_FINE_LOCATION': 'granted',
         'android.permission.READ_EXTERNAL_STORAGE': 'granted',
         'android.permission.ACCESS_MEDIA_LOCATION': 'granted' } }

### What are the steps to reproduce (after prerequisites)?
Get pictures tags location in an Android 11 device.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ❌     |
| Android |    ✅     |

## Checklist

- [X] I have tested this on a device and a simulator
- [X] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [X] I updated the typed files (TS and Flow)
- [X] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-permissions",
-  "version": "3.1.0",
+  "version": "3.2.0",
```
Comment (mikehardy) [inline package.json]:
```suggestion
  "version": "3.1.0",
```

bumping package versions is typically the domain of the maintainer (and frequently fully automated) - I'd remove this - as a maintainer on other packages I will never merge a version change in a PR personally

**Review by mikehardy — CHANGES_REQUESTED** [review]
certainly looks reasonable with the small caveat of the version change, I'd skip that personally.
awesome for posting a PR to move the feature request forward though! seriously

**Comment by zoontek** [comment]
LGTM, excepted the version change indeed

File: package.json (hunk 1 above)
Comment (hectoruiz) [reply package.json]: Sorry it's my first time collaborating in a library and I didn't know it. I'll change

**Review by mikehardy — APPROVED** [review]
A thing of beauty :-)

File: package.json (hunk 1 above)
Comment (mikehardy) [reply package.json]: no worries, most people never even make that first try. There are lots of subtle things like that, but in the end - forward progress!

**Comment by zoontek** [comment]
It's released: https://github.com/zoontek/react-native-permissions/releases/tag/3.2.0
Thanks @hectoruiz !

**Pull request merged**